provisioning_templates/provision/kickstart_default.erb

```diff
@@ -204,12 +209,19 @@ sed -e 's/DEFAULTKERNEL=kernel-uek/DEFAULTKERNEL=kernel/g' -i /etc/sysconfig/ker
 
 <%= snippet('ansible_provisioning_callback') %>
 
+touch /tmp/foreman_built
 sync
+<%= section_end -%>
 
-# Inform the build system that we are done.
-echo "Informing Foreman that we are built"
-wget -q -O /dev/null --no-check-certificate <%= foreman_url('built') %>
-) 2>&1 | tee /root/install.post.log
-exit 0
-
+<%#
+The last post section halts Anaconda to prevent endless loop on EL7+
+%>
+%post <%= erroronfail -%>
+if test -f /tmp/foreman_built; then
+  echo "calling home: build is done!"
+  wget -q -O - --body-file=/mnt/sysimage/root/install.post.log --method GET --header="Content-Type: text/xml" --no-check-certificate <%= foreman_url('built') %>
```

Comment:
Shouldn't the `Content-type` be `text/plain`?

Reply:
Yeah typo.